### PR TITLE
To do list with webpack

### PR DESCRIPTION
**Pull request by Said-laasri**
**Created** ` index.html` file using the HTML Webpack Plugin.
**Created** an `index.js` file and set an (array of objects).
**Created** a _function to iterate over the tasks array and populate an HTML list item element for each task._
**Create** a style.css and set rules for the To-Do List. CSS is loaded by Webpack Style/CSS Loader.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    index: 2,
+  },
+  {
+    description: 'read my book',
+    comleted: false,
+    index: 3,
+  },
+  {
+    description: 'working on my coding skills',
+    comleted: false,
```
Comment (DammyShittu) [inline src/index.js]: - [x] You made some typo errors here. It should be `completed`, not `comleted`. Kindly fix that

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Said-laasri,

✔️ App is set up with Webpack
✔️ There is a function of populating the HTML page

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Said-laasri) [reply src/index.js]: @DammyShittu  Thank you for reviewing my project, I will make the necessary corrections as suggested

**Review by descholar-ceo — APPROVED** [review]
Hi @Said-laasri , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Addressed the requested changes ✔️ 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Said-laasri** [comment]
@descholar-ceo  Thank you for reviewing my project.

**Pull request merged**